drivers/include/periph/rtt.h

```diff
@@ -22,6 +22,11 @@
  * On many CPUs, certain power states might need to be blocked in rtt_init(), so
```

Comment:
doccheck has issues with this line:
i am not sure if it is about this file: if so please fix, 
but i think it is about an other file (generic doxygen noise increase) (where rtt_init can be found) (in that case do not fix in this PR))